manifests/v1beta1/installs/katib-external-db/kustomization.yaml

```diff
@@ -3,28 +3,17 @@ kind: Kustomization
 namespace: kubeflow
 resources:
   # Namespace.
-  - ../../components/namespace.yaml
+  - namespace.yaml
```

Comment:
Any ideas how we can keep namespace under `manifests/v1beta1/components` ?
Maybe create folder `components/namespace` and add `namespace.yaml` and `kustomization.yaml` there.
WDYT @DavidSpek  ? cc @yanniszark

Reply:
I think this is the easiest way so I have implemented this.